spec/models/conference_spec.rb

```diff
@@ -1639,4 +1639,26 @@
       expect(free_ticket.price_cents).to eq(0)
     end
   end
+
+  describe 'after_update' do
+    let(:conference) { create(:conference) }
+    let(:first_scheduled_event) { create(:event_scheduled, program: conference.program, hour: conference.start_date + conference.start_hour.hours) }
+    let(:second_scheduled_event) { create(:event_scheduled, program: conference.program, hour: conference.start_date + conference.end_hour.hours + 1.hour) }
```

Comment:
I think it makes more sense to name the variables into something like scheduled_event_after_conference, scheduled_event_during_conference, etc.